x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/display_settings/display_settings_logic.ts

```diff
@@ -116,6 +132,10 @@ export const DisplaySettingsLogic = kea<
     setUrlField: (urlField: string) => urlField,
     setSubtitleField: (subtitleField: string | null) => subtitleField,
     setDescriptionField: (descriptionField: string) => descriptionField,
```

Comment:
In the [other PR](https://github.com/elastic/ent-search/pull/3746/files?file-filters%5B%5D=.scss&file-filters%5B%5D=.ts#diff-8de04bd9e5c5dfd7466f6dd0109779c642f56fcfe17e60a0c63df92a97b5d72fR113), this had `null` fallback type. Was the omission intended?

Reply:
Nope, great catch!